schema/deploy/util_functions/upsert_timestamp_columns.sql

```diff
@@ -93,12 +94,28 @@ begin
     execute(trigger_string);
   end if;
 
+  if not exists (select *
```

Comment:
Is it a worthwhile optimization to move the creation of this trigger inside the `if add delete = true...` block above on line 64? This way the trigger will not be created if the deleted_at column does not exist and the function will not called unnecessarily on every update for tables without a `deleted_at` column. It would also remove the need for the `if to_jsonb(old) ? 'deleted_at' then..` check in `deleted_records_are_immutable()`.

Reply:
I don't know how expensive a `to_jsonb(...) ? ...` operation is, but I like removing code 👌